RELEASE_PROCESS.md

```diff
@@ -19,36 +19,23 @@ release.
 
 In order to release, you will need the following accounts/permissions:
 
-- Local checkout of the chef-server repository
-- Push access to the chef-server github repository
 - Chef Software, Inc Slack account
 - VPN account for Chef Software, Inc.
 - Account on [https://discourse.chef.io](https://discourse.chef.io) using your Chef email address
-- Login for wilson.ci.chef.co (This is linked to your github account.)
-- Access to artifactory.chef.co
 - Access to automate.chef.co
-- The CHANGELOG_GITHUB_TOKEN environment variable set to a github token gathered
-  [here](https://github.com/settings/tokens/new?description=GitHub%20Changelog%20Generator%20token).
-  Read access to repositories and issues should be sufficient. (Check the "Full control of private repositories" box.)
-- Run `export CHANGELOG_GITHUB_TOKEN="«your-40-digit-github-token»"` to set your new token that you just created
-- Install Github Changelog Generator
-  + `gem install github_changelog_generator`
 
 :warning: If it is a Friday -- do we really need that release today? :warning:
 
 ## THE PROCESS
 
 ### Informing everyone of a pending release.
 
-- [ ] Announce your intention to drive the release to #cft-announce on slack.
-- [ ] Cross-post this intention on #chef-server and #pool.  Determine whether
-  this upgrade requires a major/minor/patch bump and what the major changes
-  are.
-- [ ] Ensure that the chef-server pipeline on wilson is currently green - you
-  cannot ship if master is broken.
```

Comment:
For other reviewers: We will now be promoting binary artifacts so the *current* state of wilson isn't an issue since wilson had to pass for the build to get into current.